Description: don't require python headers to remove build-dep on python-dev and python-gtk2-dev
Author: Yann Leboulanger <asterix@lagaule.org>
Origin: upstream,http://hg.gajim.org/gajim/diff/252bb3cf2c59/configure.ac
Last-Update: 2011-05-07

--- a/configure.ac
+++ b/configure.ac
@@ -36,25 +36,10 @@
 AM_NLS
 
 
-dnl ****
-dnl pygtk and gtk+
-dnl ****
-PKG_CHECK_MODULES([PYGTK], [gtk+-2.0 >= 2.16.0 pygtk-2.0 >= 2.16.0])
-AC_SUBST(PYGTK_CFLAGS)
-AC_SUBST(PYGTK_LIBS)
-PYGTK_DEFS=`$PKG_CONFIG --variable=defsdir pygtk-2.0`
-AC_SUBST(PYGTK_DEFS)
-
-AM_PATH_PYTHON([2.5])
-if test "x$PYTHON" = "x:"; then
-	AC_MSG_ERROR([Python not found])
-fi
-
 ACLOCAL_AMFLAGS="\${ACLOCAL_FLAGS}"
 AC_SUBST(ACLOCAL_AMFLAGS)
 
-AM_CHECK_PYTHON_HEADERS(,[AC_MSG_ERROR(could not find Python headers)])
-AC_SUBST([PYTHON_INCLUDES])
+AM_PATH_PYTHON([2.5])
 
 dnl ****
 dnl enable installation in python-2.x/site-packages/gajim
